### PR TITLE
Frederick visualisation

### LINKED ISSUE #13: Add Visualisation
As a user, I want to be able to visualize my expenses so that i can better understand where I am spending the most

### PR DESCRIPTION
**Pull request by wwweert123**
Fix #13

### CODE REVIEW

**Review by ryan1604 — APPROVED** [review]
LGTM

**Review by NeoMinWei — APPROVED** [review]
LGTM

**Review by hshiah — APPROVED** [review]
LGTM

File: src/main/java/seedu/financialplanner/visualisations/Categorizer.java
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/visualisations/Categorizer.java:1]: good job on using hashmaps

**Review by NeoMinWei — COMMENTED** [review]
LGTM

**Pull request merged**